### ISSUE TITLE
following the reasoning in the paths of the UI

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#263]
Doing the "math" as Andy calls it of the reasoning is cumbersome for users.  We're not sure if this is something the UI can enhance.

**Comment by sandrine-m** [comment]
@sstemann  is this issue linked with #282 and #342 or is that a completely different issue? I am trying to move along this "no status ticket". Thank you!

**Comment by gglusman** [comment]
I read this as being closer to #342, but it's a wider issue: it would be great if Translator had much stronger ways to _explain the provenance_ of each assertion.

- When a KP reports an assertion drawn from a database, it's quite sufficient to link out to that database, ideally to a page in that database that includes that assertion (not the homepage/search).
- When a KP reports a text-mined assertion, linking to the paper is great, but frequently insufficient as the user may have a hard time finding how the paper connects to the assertion. It would be great to be able to provide a dynamic page that explains it fully, including important bits like differences between the identifier used in the assertion and the actual name used in the paper.
- When a KP reports an assertion computed from raw data, it would be great to offer a dynamic page explaining how the specific assertion was computed from the raw data, with details on cohort size and such.
- Finally and to Sarah's point, when an ARA reports a path it constructed, it would be useful to have a dynamic page explaining to the user the logic rule(s) used to do that reasoning.
To clarify, when I write 'a dynamic page', I mean a page that is tailored to the specific example being explained, as opposed to a page that explains what the KP/ARA does in generic terms.

**Comment by sandrine-m** [comment]
Thank you @gglusman ! Those are great points! If I can add to your first point, it could be nice to be able to link in a user readible format to the definitions of our categories and predicates from the biolink model as a way to express possible discrepencies in what is reported by a source and what predicate maps to this definition. That would _explain_ (provenance) the choice of vocabulary presented to the user and internally represented.

**Comment by gprice1129** [comment]
@sstemann @gglusman @sandrine-m we are not sure if this ticket has changed from the initial intent or what the feature request actually is. Please advise.

**Comment by sierra-moxon** [comment]
@gprice1129 - is there any priority being discussed for this issue in the UI team?

**Comment by gprice1129** [comment]
@sierra-moxon yes, we have discussed this quite a bit. We will be working on this in the future, but it probably won't be implemented by Octopus.
